### ISSUE TITLE
mlprimitives/candidates/timseries.py lacks ability to return sequences of desired length

### ISSUE THREAD

**Issue by itinawi** [HDI-Project/MLPrimitives#83]
I want to push all the final touches and changes that were made after MLPrimitives release v0.1.4 in order to make the LSTM pipeline primitives work as expected, since there are bugs that prevent them from being used now.

I will push the changes to the primitives that are currently in the working demo as tested on private servers and repos.

**Comment by csala** [comment]
@itinawi would you mind being a bit more specific about the affected primitives and the required changes?

**Comment by itinawi** [comment]
@csala Here are the changes, in more detail:

`mlprimitives/candidates/timseries.py`: change `y` such that we return sequences. I will add another commit now so that, based on a hyperparameter called `return_sequences` we return either a single point or a sequence. This will be in another pull request.

`mlprimitives/candidates/timeseries_errors.py`: fixed a problem in which the predicted anomalies had a negative index.

`mlprimitives/jsons/keras.Sequential.LSTMTimeseriesRegressor.json`: made the model have one LSTM, one dropout, and one dense output layers.

`mlprimitives/jsons/mlprimitives.candidates.timeseries.rolling_window_average.json` and `/time_segments_average.json`: added `time_column` and `value_column` as parameters so the MLBlocks context works.

**Comment by csala** [comment]
@itinawi the changes that you request look right, but I think that it would be much clearer to resolve them in separated issues, so I will now create new issues and rename this one.

**Comment by itinawi** [comment]
This issue addresses only the following problem:
`mlprimitives/candidates/timseries.py`: change `y` such that we return sequences. The `return_sequences` method needs a feature such that based on a hyperparameter called `return_sequences` we return either a single point or a sequence.

**Comment by csala** [comment]
New issues #89 #90 and #91 have been opened

**Comment by itinawi** [comment]
The `flatten` flag issue will be added as its own primitive. 

This issue will focus on the `return_sequences` problem. 

Instead of returning the single next point `y_{t+1}`, it would be better to give the user the ability to specify the length of the interval. 

The function signature would become:
`def rolling_window_sequences(X, x_window_size, y_window_size, value_column, time_column)`

The new variables:
- `x_window_size`: the size of the window that `x` will have. This would be the input to any model that will try to predict `y` based on the `x_{t-x\_window\_size}` ... `x_{t}` values.
- `y_window_size`: the size of the window that `y` will have. This is how much steps further the model will predict `y_{t+1}` to `y_{t+1+y\_window\_size}`

**Comment by csala** [comment]
This functionality has already been covered in #84, so this issue is not needed anymore.
